test/release/examples/primers/arrays.chpl

```diff
@@ -157,9 +157,10 @@ proc negateAndPrintArr(X: [?D] real) {
 negateAndPrintArr(B);
 
 //
-// Arrays are passed to routines by reference (``ref``) by default, so
-// the modifications to ``X`` in procedure ``negateAndPrintArr()`` are
-// reflected back in the actual argument ``B`` as well:
+// Arrays are passed to routines by constant reference (``const ref``) by
+// default, so for the modification to ``X`` in procedure
+// ``negateAndPrintArr()`` to be legal and reflected back to the actual argument
+// ``B``  the explicit ``ref`` must be used.
```

Comment:
I propose:

```
// Arrays are passed to a routine by constant reference (``const ref``) by
// default, which does not allow them to be modified within the routine.
// Therefore the above procedure ``negateAndPrintArr()`` uses a non-constant
// reference intent ``ref`` explicitly, so that its modifications of ``X``
// are allowed and reflected back in the actual argument ``B``:
```

Reply:
I have changed it to the following, which is slightly tweaked:

```chapel
// Arrays are passed to routines by constant reference (``const ref``) by
// default, which does not allow them to be modified within the routine.
// The above procedure ``negateAndPrintArr()`` must use a non-constant
// reference intent (``ref``) explicitly, so that its modifications of ``X``
// are both allowed and reflected in the actual argument ``B``:
```